src/Commands/ImportBrands.php

```diff
@@ -40,7 +40,7 @@ public function handle(StatamicEntryAction $statamicEntryAction): int
                     array_merge([
                         'locale' => $site->handle(),
                         'site' => $site->handle(),
-                    ], ...Event::dispatch('rapidez-statamic:brand-entry-data', ['brand' => $brand]))
+                    ], ...[Eventy::filter('rapidez.statamic.brand.entry.data', $brand)])
```

Comment:
In this case destructuring wouldn't be needed anymore
```suggestion
                    ], Eventy::filter('rapidez.statamic.brand.entry.data', $brand))
```
Does everything still function the same as before? 
Can more data still be added to these fields without having to overwrite the original array of fields?

If not, it may be more beneficial to wrap the original `Event::dispatch` method in an array_filter to remove null and falsey values

Reply:
The functionality is not exactly the same as before. Because of this I took your advice to array_filter the data from the event. Did this in a new PR: #51 because almost everything in this PR would be irrelevant and the name of the commits wouldn't match the change.